app/uk/gov/hmrc/exports/scheduler/jobs/emails/SendEmailsJob.scala

```diff
@@ -97,20 +97,29 @@ class SendEmailsJob @Inject() (
 
       case Success(MissingData) =>
         logger.warn(s"Could not send email because some data is missing")
-        failSingle(workItem)
+        failOrCancelSingle(workItem)
 
       case Success(BadEmailRequest(msg)) =>
         logger.warn(s"Email service returned Bad Request response: [$msg]")
-        failSingle(workItem)
+        failOrCancelSingle(workItem)
 
       case Success(InternalEmailServiceError(msg)) =>
         logger.warn(s"Email service returned Internal Service Error response: [$msg]")
         logger.warn(s"Will try again in ${interval.toMinutes} minutes")
```

Comment:
This might no longer be true, if the failure count equals 9 then it won't be tried again!